src/DynamoCore/Graph/Nodes/CustomNodes/Function.cs

```diff
@@ -309,9 +309,9 @@ private void ValidateDefinition(CustomNodeDefinition def)
         ///     Validates passed Custom Node definition and synchronizes node with it.
         /// </summary>
         /// <param name="def">Custom Node definition.</param>
-        public void ResyncWithDefinition(CustomNodeDefinition def)
+        public void ResyncWithDefinition(CustomNodeDefinition def, bool removeErrorState = true)
```

Comment:
you can't change a public method signature at this point, even adding a default argument is an API breaking change.

Reply:
you can add a new overload and obsolete this method.